bin/dbt-create.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+import os
+DBT_ROOT=os.environ["DBT_ROOT"]
+exec(open(f'{DBT_ROOT}/scripts/dbt_setup_constants.py').read())
+
+import sys
+sys.path.append(f'{PROD_BASEPATH}/dunedaq-v2.8.2/dbt-pyvenv/lib/python3.8/site-packages')
```

Comment:
Any python modules you need from an existing instance of `dbt-pyvenv`?

This introduces the requirement that the user have access to an existing release. This might be problematic for CI jobs using containers with no frozen/nightly releases installed.